larpandoracontent/LArHelpers/LArPcaHelper.cc

```diff
@@ -18,17 +18,29 @@ namespace lar_content
 
 void LArPcaHelper::RunPca(const CaloHitList &caloHitList, CartesianVector &centroid, EigenValues &outputEigenValues, EigenVectors &outputEigenVectors)
 {
-    CartesianPointVector pointVector;
+    CartesianPointDoublePairVector cartesianPointDoublePairVector;
 
     for (const CaloHit *const pCaloHit : caloHitList)
-        pointVector.push_back(pCaloHit->GetPositionVector());
+        cartesianPointDoublePairVector.push_back(std::make_pair(pCaloHit->GetPositionVector(), 1.0));
```

Comment:
As discussed, create LArObjectHelper with just the existing LArPfoHelper::TypeAdaptor in initially. Obviously, then remove this from LArPfoHelper.

Reply:
Done and testing pca now.